test/remote/db_tests.js

```diff
@@ -661,6 +662,16 @@ describe('remote db', function() {
     }
   )
 
+  afterEach(() => {
+    return db.emailRecord(ACCOUNT.email)
+      .then(function (emailRecord) {
+        return db.deleteAccount(emailRecord)
+      })
+      .catch(function (err) {
+        //
```

Comment:
Yea, I'd probably rather remove this catch, since if an error happened, we want to know.